js/src/services/human-resource-api.ts

```diff
@@ -96,7 +100,10 @@ export const applyPosition = ({
       success: true,
       data: null,
     }))
-    .catch<Failure>(() => ({ success: false, reason: "Error" }));
+    .catch<FormFailure>((resp) => ({
+      success: false,
+      errors: resp?.response?.data?.errors || [],
+    }));
 
 export const applySalary = ({ userId, ...body }: ApplySalaryRequestParams) =>
```

Comment:
Why use Spread for body?

Reply:
because all params are a single object. one property userId I need in the url and other props - amount and comment in body.